hackett-lib/hackett/private/adt.rkt

```diff
@@ -285,6 +289,8 @@
          (values #`(#%type:wobbly-var #,a^) '() #{values #'_ %}))]
       [(pat-str _ str)
        (values (expand-type #'String) '() #{values str %})]
+      [(pat-int _ int)
+       (values (expand-type #'Integer) '() #{values int %})]
```

Comment:
I think it makes sense to put the `force-` here instead of in the parsing logic, so that `pat-int` and `pat-str` just contain the raw values. Otherwise, this looks good to me.

Reply:
Can you say more about how I should do that? I tried changing `#{values int` to `#{values (force- int)` and got an error that `force-` is unbound at phase 1. Should I add a require, or is that the wrong place to force?

Reply:
I think you probably want `#{values #`(app force- #,int) %}`, since `int` is syntax.